.github/workflows/unit-test.yml

```
@@ -1,35 +1,42 @@
+      run: npm install
+    - name: Test
+      run: npm run unit_test:ci
+      env:
+        GH_UNIT_TEST_ENV: true
+        BRAND_NAME: "Fyle"
+        WEBPAGE_TITLE: "Fyle Integrations Settings"
+        BRAND_ID: "fyle"
+        SUPPORT_EMAIL: "staging-1-in"
+        HELP_ARTICLE_DOMAIN: "support@fylehq.com"
+        ENV_ID: "https://help.manageexpenses.capitalone-fylehq.com"
+        FYLE_CLIENT_ID: "lolo" 
```

yamllint

[error] trailing spaces

(trailing-spaces)